packages/go/analysis/azure/post.go

```diff
@@ -272,6 +273,7 @@ func roleMembers(tx graph.Transaction, tenantRoles graph.NodeSet, additionalRela
 		} else {
 			// TODO: This could be more optimal by iterating in place instead of aggregating all results
 			members.AddSet(paths.AllNodes())
+			members.Remove(tenantRole.ID)
```

Comment:
I'm not seeing anything obvious, but I just wanted to make sure nothing calling `RoleMembers()` or `RoleMembersWithGrants()` depends on the Role ID being included in the returned list of members.

Reply:
The updated logic looks good!